floodviz/posterior_build/hydro_thumbnail.js

```diff
@@ -0,0 +1,248 @@
+
```

Comment:
Could use comments indicating this is same as hydrograph code and what differences are.